qiita_pet/templates/study_ajax/base_info.html

```diff
@@ -1,11 +1,50 @@
+<link rel="stylesheet" href="/static/vendor/css/select2.min.css" type="text/css">
+<script type="text/javascript" src="/static/vendor/js/select2.min.js"></script>
+<script type="text/javascript" src="/static/js/sharing.js"></script>
+<script type="text/javascript">
+$(document).ready(function () {
+	current_study = {{study_info['study_id']}};
+	update_share();
+});
+</script>
 <p style="font-weight:bold;">Abstract</p>
 <p>{{study_info['study_abstract']}}</p>
 Study ID:  {{study_info['study_id']}}<br />
 Publications: {% raw publications %}<br />
 PI: {% raw pi %}<br />
 Lab Contact: {% raw contact %}<br />
+Shared With: <span id="shared_html_{{study_info['study_id']}}">
+</span><br/>
 Samples: {{study_info['num_samples']}}<br />
+{% if share_access %}
+<a class="btn btn-default" data-toggle="modal" data-target="#share-study-modal-view" onclick="modify_sharing({{study_info['study_id']}});"><span class="glyphicon glyphicon-share"></span> Share</a>
+{% end %}
 {% if editable %}
   <a class="btn btn-default" href="/study/edit/{{study_info['study_id']}}"><span class="glyphicon glyphicon-edit"></span> Edit</a>
   <a class="btn btn-danger" href="#" data-toggle="modal" data-target="#delete-study"><span class="glyphicon glyphicon-trash"></span> Delete</a>
-{% end %}
+
+ <!-- Modal used to share the study -->
+<div class="modal fade" id="share-study-modal-view" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+        <h4 class="modal-title" id="myModalLabel">Modify Sharing Settings</h4>
+      </div>
+        <div class="modal-body">
+          <div>
+            <div class="form-group">
+              <label for="shares-select">Add/Remove Users</label>
+              <select multiple id="shares-select" style="width:50%"></select>
+              <br>
+              <br>
+              Adding or removing email addresses automatically updates who the study is shared with. Once you click the `X` or give mouse focus to the study page you'll see your new sharing settings.
```

Comment:
Note that we don't have Markdown rendering here so \`X\` will just show as usual, maybe worth using some other HTML tag to highlight this if needed or just remove it. Also automatically seems redundant in this sentence.